src/content/language/functions.md

```diff
@@ -620,3 +618,55 @@ is not `final`) an external setter.
 [conditional expression]: /language/operators#conditional-expressions
 [Flutter]: {{site.flutter}}
 [trailing commas]: /language/collections#lists
+
+## Getters and setters
```

Comment:
This section looks great. Can you go through the markdown file and make sure that all lines are under 80 characters? There should be a setting in VS Code that let's you track when something goes over that line. If you need help finding it, let me know.